src/perform_step/low_order.jl

```diff
@@ -229,3 +229,127 @@ end
+      end
+      mil_correction += ggprime*@view(WikJ[:,i])
+    end
+    if integrator.opts.adaptive
+      K = @.. uprev + dt*du₁
+      u = K + L*dW + mil_correction
+    else
+      u = uprev + dt*du₁ + L*dW + mil_correction
+    end
+  end
+
+  if integrator.opts.adaptive
+    du₂ = integrator.f(K,p,t+dt)
+    if typeof(dW) <: Number || is_diagonal_noise(integrator.sol.prob)
+      tmp  = dt*(du₂ -du₁)/2
+      En = W.dW.^3 .* ((du₂-L)/(integrator.sqdt)).^2 / 6
+      tmp = calculate_residuals(tmp, En, uprev, u, integrator.opts.abstol,
+            integrator.opts.reltol, integrator.opts.delta, integrator.opts.internalnorm, t)
+    else
+      En = integrator.opts.internalnorm(W.dW,t)^3*ggprime_norm^2 / 6
+      tmp = @.. integrator.opts.internalnorm(dt*(du₂ - du₁)/2,t)
+      tmp = @.. calculate_residuals(tmp, En, uprev, u, integrator.opts.abstol,
+                integrator.opts.reltol, integrator.opts.delta, integrator.opts.internalnorm, t)
+    end
+    integrator.EEst = integrator.opts.internalnorm(tmp, t)
+  end
+  integrator.u = u
+end
+
+@muladd function perform_step!(integrator,cache::RKMil_GeneralCache,f=integrator.f)
+  @unpack du₁, du₂, K, tmp, ggprime, L, mil_correction = cache
+  @unpack t,dt,uprev,u,W,p = integrator
+  dW = W.dW; sqdt = integrator.sqdt
+  Wik = cache.WikJ
+  get_iterated_I!(dt, dW, Wik, integrator.alg.c)
+  WikJ = Wik.WikJ
+
+  integrator.f(du₁,uprev,p,t)
+  integrator.g(L,uprev,p,t)
+  @.. mil_correction = zero(eltype(u))
+  ggprime_norm = zero(eltype(ggprime))
+
+  if alg_interpretation(integrator.alg) == :Ito
+    if typeof(dW) <: Number || is_diagonal_noise(integrator.sol.prob)
+      @.. WikJ -= 1//2*dt
+    else
+      WikJ .-= 1//2 .* dt .* Eye{eltype(dW)}(length(dW))
+    end
+  end
+
+  if typeof(dW) <: Number || is_diagonal_noise(integrator.sol.prob)
+    @.. K = uprev + dt*du₁
+    @.. du₂ = zero(eltype(u))
+    tmp .= (alg_interpretation(integrator.alg) == :Ito ? K : uprev) .+ integrator.sqdt .* L
+    integrator.g(du₂,tmp,p,t)
+    @.. ggprime = (du₂ - L)/sqdt
+    ggprime_norm = integrator.opts.internalnorm(ggprime,t)
+    @.. u = K + L*dW + ggprime*WikJ
+  else
+    for i in 1:length(dW)
+      @.. K = uprev + dt*du₁ + sqdt*@view(L[:,i])
+      integrator.g(ggprime, K, p, t)
+      @.. ggprime = (ggprime - L)/sqdt
+      if integrator.opts.adaptive
+        ggprime_norm += integrator.opts.internalnorm(ggprime,t)
+      end
+      mul!(tmp,ggprime,@view(WikJ[:,i]))
+      @.. mil_correction += tmp
+    end
+    mul!(tmp,L,dW)
+    if integrator.opts.adaptive
+      @.. K = uprev + dt*du₁
+      @.. u = K + tmp + mil_correction
+    else
+      @.. u = uprev + dt*du₁ + tmp + mil_correction
+    end
+  end
+
+  if integrator.opts.adaptive
+      En = integrator.opts.internalnorm(W.dW,t)^3*ggprime_norm^2 / 6
+      integrator.f(du₂,K,p,t+dt)
+      @.. tmp = integrator.opts.internalnorm(integrator.opts.delta * dt * (du₂ - du₁) / 2,t) + En
```

Comment:
Same here.

Reply:
This error estimate is taken from `RKMil_Commute` and `RKMil` So I think it should work here too.

Reply:
And also this will change the type of `tmp`.